### PR TITLE
[FEMDEM] Some minor improvements

### PR DESCRIPTION
**Pull request by AlejandroCornejo**
that's it

### CODE REVIEW

File: applications/FemToDemApplication/custom_elements/generic_small_strain_femdem_element.cpp [hunk 1]
```diff
@@ -105,7 +105,11 @@ void GenericSmallStrainFemDemElement<TDim,TyieldSurf>::InitializeSolutionStep(
     ProcessInfo& rCurrentProcessInfo
     )
 {
-	this->ComputeEdgeNeighbours(rCurrentProcessInfo);
+	if (this->GetValue(RECOMPUTE_NEIGHBOURS) > 0) {
```
Comment (loumalouomega) [inline applications/FemToDemApplication/custom_elements/generic_small_strain_femdem_element.cpp]: Is this an integer or boolean?

File: applications/FemToDemApplication/fem_to_dem_application_variables.cpp [hunk 2]
```diff
@@ -39,7 +39,7 @@ namespace Kratos
 	KRATOS_CREATE_VARIABLE(int, INTEGRATION_COEFFICIENT)
 	KRATOS_CREATE_VARIABLE(std::string, MAPPING_PROCEDURE)
 	KRATOS_CREATE_VARIABLE(bool, GENERATE_DEM)
-	KRATOS_CREATE_VARIABLE(int, RECOMPUTE_NEIGHBOURS)
+	KRATOS_CREATE_VARIABLE(bool, RECOMPUTE_NEIGHBOURS)
```
Comment (loumalouomega) [inline applications/FemToDemApplication/fem_to_dem_application_variables.cpp:42]: AHA!!

File: applications/FemToDemApplication/custom_elements/generic_small_strain_femdem_element.cpp (hunk 1 above)
Comment (loumalouomega) [reply applications/FemToDemApplication/custom_elements/generic_small_strain_femdem_element.cpp]: Check is true and set False

Comment (loumalouomega) [inline applications/FemToDemApplication/custom_elements/generic_small_strain_femdem_element.cpp]:
```suggestion
	if (this->GetValue(RECOMPUTE_NEIGHBOURS)) {
```

File: applications/FemToDemApplication/custom_elements/generic_small_strain_femdem_element.cpp [hunk 3]
```diff
@@ -105,7 +105,11 @@ void GenericSmallStrainFemDemElement<TDim,TyieldSurf>::InitializeSolutionStep(
...
+		this->ComputeEdgeNeighbours(rCurrentProcessInfo);
+		this->SetValue(RECOMPUTE_NEIGHBOURS, 0);
```
Comment (loumalouomega) [inline applications/FemToDemApplication/custom_elements/generic_small_strain_femdem_element.cpp]:
```suggestion
		this->SetValue(RECOMPUTE_NEIGHBOURS, false);		
```

**Review by loumalouomega — CHANGES_REQUESTED** [review]
Minor comments

**Comment by AlejandroCornejo** [comment]
changes done @loumalouomega

**Review by loumalouomega — APPROVED** [review]
👌

**Pull request merged**